pkg/ceph/rgw/objectstore_test.go

```diff
@@ -114,11 +127,17 @@ func TestDeleteStore(t *testing.T) {
 	}
 	context := &Context{context: &clusterd.Context{Executor: executor}, Name: "myobj", ClusterName: "ns"}
 
+	// Delete one of two object stores
 	err := DeleteObjectStore(context)
 	assert.Nil(t, err)
-	assert.Equal(t, 6, poolsDeleted)
-	assert.Equal(t, 6, rulesDeleted)
+	expectedPoolsDeleted := 5
+	if expectedDeleteRootPool {
+		expectedPoolsDeleted++
+	}
+	assert.Equal(t, expectedPoolsDeleted, poolsDeleted)
+	assert.Equal(t, expectedPoolsDeleted, rulesDeleted)
```

Comment:
its expected that there is 1 rule for each pool i assume?

Reply:
yes, that's correct